backend/LexBoxApi/Controllers/CrdtController.cs

```diff
@@ -85,4 +87,13 @@ public async Task<ActionResult<Guid>> GetProjectId(string code)
 
         return Ok(projectId);
     }
+
+    [HttpPost("crdt-sync/{projectId}")]
+    [AdminRequired]
```

Comment:
Is this perhaps a start for handling long syncs?
```suggestion
    [AdminRequired]
    [RequestTimeout(300_000)]
```

Reply:
By the way, I suggested 300s, because that's apparently the default `fetch()` timeout in Chrome.
In Firefox, it's apparently only 90s.

https://stackoverflow.com/a/62470009/2301416